pkgs/applications/video/obs-studio/plugins/obs-browser-transition.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  obs-studio,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-browser-transition";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "exeldro";
+    repo = "obs-browser-transition";
+    rev = version;
+    sha256 = "sha256-m5UDqnqipkybXAZqS7c2Sj/mJKrDBkXElyc0I+c1BmE=";
```

Comment:
```suggestion
    tag = version;
    hash = "sha256-m5UDqnqipkybXAZqS7c2Sj/mJKrDBkXElyc0I+c1BmE=";
```

Reply:
Pushed an update to address the feedback.